src/snowflake/snowpark/functions.py

```diff
@@ -4594,6 +4595,90 @@ def dayofyear(e: ColumnOrName) -> Column:
     return builtin("dayofyear")(c)
 
 
+def window(
+    timeColumn: ColumnOrName,
+    windowDuration: str,
+    slideDuration: Optional[str] = None,
+    startTime: Optional[str] = None,
```

Comment:
camelCase is generally not pythonic. Can we convert this into snake_case

Reply:
I agree, but to maintain compatibility with the pyspark implementation I think camelCase might be required. I could modify so that either version is accepted.

Reply:
IHMO api style consistency within the snowpark lib outweighs consistency with pyspark.

Reply:
yup. the kwarg conversion can be handled by conversion tools